filters_test.go

```diff
@@ -189,6 +189,27 @@ func TestExecFilter(t *testing.T) {
 	if out != "" {
 		t.Error("want exec on erroneous pipe to be a no-op, but it wasn't")
 	}
+
+	// Testing cases with double quotes wrapped in single quoutes.
+	testPipe := NewPipe()
+	testQuotes := []struct {
+		testStrings string
+		want        string
+	}{
+		{`echo 'test'`, "test"},
```

Comment:
Won't this only work if the `echo` command exists and is in the user's path? Ideally, we wouldn't need to depend on any userland tools for `script` to work; that's part of the reason for writing it. If we just want to test that we correctly assemble command line arguments for a given command, we don't need to actually run the command to do that, do we?